routes/job.js

```diff
@@ -28,4 +32,72 @@ router.put('/', Utils.asyncRoute(async function (req, res, next) {
     res.send(room);
 }));
 
+router.get('/:jobId', Utils.asyncRoute(async function (req, res, next) {
+    const {jobId} = req.params;
+    const connector = new DbConnector();
+    try {
+        connector.connect();
+        const siteModel = await SiteModel.findOne({'rooms.jobs._id': mongoose.Types.ObjectId(jobId)}, {'rooms.jobs.$': true})
+        .sort({timestamp: 1}).exec();
+        const job = siteModel.rooms[0].jobs.find(job => job._id.equals(jobId));
+        
+        res.send(job);
+    } catch (e) {
+        console.error(e);
+        res.status(500).send(e);
+    }
+}));
+
+
+router.post('/analysis/:jobId', Utils.asyncRoute(async function (req, res, next) {
+    const {jobId} = req.params;
+    const {image, modelPrediction, analysisQueueId} = req.body
+    const connector = new DbConnector();
+    const timestamp = new Date().getTime();
+    const jobAnalysis = new JobAnalysis({
+        jobId: jobId,
+        timestamp: timestamp,
+        modelPrediction: modelPrediction,
+        image: image,
```

Comment:
JobAnalysis is missing the `imageThumbnail` field. You can probably perform the image transforms on the `image` within the object constructor.